ui/test/nuxeo-uploader-behavior.test.js

```diff
@@ -16,20 +16,20 @@ limitations under the License.
 */
 import { fixture, html, login } from '@nuxeo/nuxeo-elements/test/test-helpers.js';
 import '../widgets/nuxeo-file.js';
-import { UploaderBehavior } from '../widgets/nuxeo-uploader-behavior.js';
+import { UploaderBehaviorImpl } from '../widgets/nuxeo-uploader-behavior.js';
```

Comment:
nit: using `UploaderBehavior` as a alias would probably make this easier to read:
```suggestion
import { UploaderBehaviorImpl as UploaderBehavior } from '../widgets/nuxeo-uploader-behavior.js';
```